app/services/preserved_object_handler.rb

```diff
@@ -73,33 +73,32 @@ def check_existence
 
           raise_rollback_if_pc_po_version_mismatch(pres_copy.version, pres_object.current_version)
 
-          ran_moab_validation = false
-          if incoming_version == pres_copy.version
+          incoming_vers_eq_pc_vers = (incoming_version == pres_copy.version)
+          incoming_vers_gt_pc_vers = (incoming_version > pres_copy.version)
+          found_expected_version = (incoming_vers_eq_pc_vers || incoming_vers_gt_pc_vers)
+          check_status_if_not_ok_or_unexpected_version(pres_copy, found_expected_version)
```

Comment:
what if you put `check_status_if_not_ok_or_unexpected_version(pres_copy, true)` (or false)  in each of the if clauses below?   I have a preference for actually seeing "incoming_version > pres_copy.version" over the variable.